.github/workflows/broken_links_checker.yml

```diff
@@ -4,13 +4,24 @@ on:
   schedule:
     - cron: "0 5 * * *"
   push:
+    branches:
+      - main
```

Comment:
Hm, thus you will notice broken links only after the integration of a PR to `main`.